src/quo2/components/settings/data_item/view.cljs

```diff
@@ -0,0 +1,91 @@
+(ns quo2.components.settings.data-item.view
+  (:require [quo2.theme :as quo.theme]
+            [react-native.core :as rn]
+            [quo2.components.icon :as icons]
+            [quo2.components.settings.data-item.style :as style]
+            [quo2.components.markdown.text :as text]
+            [quo2.foundations.colors :as colors]
+            [quo2.foundations.resources :as quo2.resources]
+            [react-native.fast-image :as fast-image]
+            [quo2.components.avatars.account-avatar.view :as account-avatar]
+            [quo2.components.list-items.preview-list :as preview-list]
+            [quo2.components.common.not-implemented :as not-implemented]))
+
+(defn- render-right-side
+  [label icon-right? icon-color]
+  [rn/view
+   {:style style/right-container}
+   (case label
+     :preview [preview-list/preview-list
+               {:type      :user
+                :size      24
+                :list-size 3}]
+     :graph   [text/text "graph"]
+     :none    nil)
+   (when icon-right?
+     [rn/view
+      {:style {:margin-left (if (or (= label :graph) (= label :none)) 12 8)}}
+      [icons/icon
+       (if (= :none label)
+         :i/copy
+         :i/chevron-right)
+       {:color icon-color
+        :size  20}]])])
+
+(defn- render-left-side
+  [theme title status size blur? description icon subtitle label icon-color]
+  [rn/view
+   [rn/view
+    {:style style/title-container}
+    [text/text
+     {:weight :regular
+      :size   :paragraph-2
+      :style  (style/title theme)}
+     title]
+    (when (and (= :graph label) (not= :small size))
+      [text/text
+       {:weight :regular
+        :size   :label
+        :style  (style/title theme)}
+       "Days"])]
+   (if (= status :loading)
+     [rn/view {:style (style/loading-container size theme blur?)}]
+     [rn/view
+      {:style style/subtitle-container}
+      (when (not= :small size)
+        [rn/view {:style (style/subtitle-icon-container description)}
+         (case description
+           :icon    [icons/icon icon {:size 16 :color icon-color}]
+           :account [account-avatar/view
+                     {:customization-color (get-in colors/customization
+                                                   [:yellow (if (= theme :dark) 60 50)])
```

Comment:
Maybe add this in a let binding to improve readability

Reply:
also to point out here, we rarely ever want to directly set the `customization-color` like this, i.e `:yellow` -> this should be a prop `customization-color` like most other quo2 components. 
There are a few exceptions to this, where for example we are using semantic names like `:success` and `:danger` but these are not in the colors map `colors/customization` so it mostly obvious enough in the designs.


Whats more is that we also have helper functions which should be used here as there is some other pieces to handling the custom-colors.
In this case here the best approach is (imo)
```
 (colors/theme-colors 
 (colors/custom-color customization-color 50)
 (colors/custom-color customization-color 60)
 theme)                                    
```

Reply:
Done